java/bogdana_maksiuta/src/com/zoolatech/lecture3/tasks/_3/Circle.java

```diff
@@ -0,0 +1,45 @@
+package com.zoolatech.lecture3.tasks._3;
+
+import java.util.Objects;
+
+public non-sealed class Circle implements Shape {
+    final double PI = 3.1415926536;
+    int radius;
+
+    public Circle(int radius) {
+        this.radius = radius;
+    }
+
+    @Override
+    public void findPerimeter() {
+        int perimeter = (int) (radius * 2 * PI);
+        System.out.println("The perimeter of the circle is " + perimeter);
+    }
+
+    @Override
+    public void findArea() {
+        int area = (int) (Math.pow(radius, 2) * PI);
+        System.out.println("The area of the circle is " + area);
+    }
+
+    @Override
+    public String toString() {
+        return "Circle: " +
+                "PI = " + PI +
```

Comment:
`PI` will be the same for all classes, you should be only interested in radius

Reply:
fixed. Missed this moment